src/lib/specs-crawler.js

```diff
@@ -345,7 +345,16 @@ async function crawlList(speclist, crawlOptions) {
         return result;
     }
 
-    const crawlQueue = new ThrottledQueue(4);
+    const crawlQueue = new ThrottledQueue({
+        maxParallel: 4,
+        sleepInterval: origin => {
```

Comment:
isn't this missing the integration of `sleepInterval` in `ThrottledQueue`?

Reply:
Ah, yes, better with all updates :) I force pushed things.